python/services/dataservice/dmod/dataservice/__main__.py

```diff
@@ -38,7 +38,7 @@ def _handle_args():
     parser.add_argument('--object-store-host',
                         help='Set hostname for connection to object store',
                         dest='obj_store_host',
-                        default='minio_proxy')
+                        default='minio-proxy')
```

Comment:
I'm just thinking about this now, but maybe something like checking for an environment variable with a fallback for this value might be helpful. Changing '_' to '-' isn't a huge change, but I've seen it elsewhere. It being a default host (probably a docker container, right?), it might help to have this string be in a central location so that all of these types of variables may be consistent. Plus, overwriting them with environment variables would be easy enough.

Not enough to derail the PR, though. Just an early morning thought.

Reply:
I have typically tried to use args directly for external parameters.  In my mind, it makes packages less brittle.  

My real mistake may be providing the currently-correct value as a default here and most of the other analogous places for this value, as it is kind of centrally located already.  A DMOD _.env_ variable is set up to be the source of this, which passes through to (parts of) the Docker config, which passes through the entrypoint script, which provides this arg value.   I could take the defaults out and force the DMOD config item to be in place.  But even then, I can't centralize quite everything (e.g., the nginx configs).